### ISSUE TITLE
Fix parsigex with `--p2p-fuzz`

### ISSUE THREAD

**Issue by dB2510** [ObolNetwork/charon#2517]
# 🐞 Bug Report

### Description

While doing peer to peer fuzzing we found out that on enabling p2p-fuzz on one node of a 3 of 4 cluster, the cluster failed to attest. This is unexpected. It seems like parsigex breaks if any peer sends malicious data.

## 🔬 Minimal Reproduction

Run a 3 of 4 cluster with 3 nodes with normal configuration and one node with the following command:
```
charon unsafe run --p2p-fuzz (... Rest of the flags)
```
OR
Run with compose cluster:
```
compose new --build-local --p2p-fuzz && compose auto
```

## 🔥 Error

<img width="1724" alt="Screenshot 2023-08-07 at 2 02 36 PM" src="https://github.com/ObolNetwork/charon/assets/37813203/8b201ab9-3b08-425b-a2da-ae108cd3ae41">
<img width="1623" alt="Screenshot 2023-08-07 at 2 02 58 PM" src="https://github.com/ObolNetwork/charon/assets/37813203/01e62d62-a64b-4821-99ae-cf3acc583eab">

**Comment by corverroos** [comment]
I do not think this is an issue with parsigex or p2p-fuzz. I think lighthouse VC just isn't working in compose. When I run `compose new --build-local --validator-types=lighthouse && compose auto` no attestation partial signatures are submitted by any VC so not attestations are successful.

**Comment by dB2510** [comment]
@corverroos yeah you are right, this issue is a false alarm because of lighthouse VC being incompatible in compose. Will fix that next as part of #2138. Closing this one.
